packages/dev/core/src/Misc/khronosTextureContainer2.ts

```diff
@@ -50,6 +52,161 @@ function applyConfig(urls: typeof KhronosTextureContainer2.URLConfig): void {
     }
 }
 
+/**
+ * Class that defines the default KTX2 decoder options.
+ *
+ * This class is useful for providing options to the KTX2 decoder to control how the source data is transcoded.
+ */
+export class DefaultKTX2DecoderOptions {
+    private _isDirty = true;
+
+    /**
+     * Gets the dirty flag
+     */
+    public get isDirty() {
+        return this._isDirty;
+    }
+
+    private _useRGBAIfASTCBC7NotAvailableWhenUASTC?: boolean;
+    /**
+     * force a (uncompressed) RGBA transcoded format if transcoding a UASTC source format and ASTC + BC7 are not available as a compressed transcoded format
+     */
+    public get useRGBAIfASTCBC7NotAvailableWhenUASTC() {
+        return this._useRGBAIfASTCBC7NotAvailableWhenUASTC;
+    }
+
+    public set useRGBAIfASTCBC7NotAvailableWhenUASTC(value: boolean | undefined) {
+        if (this._useRGBAIfASTCBC7NotAvailableWhenUASTC === value) {
+            return;
+        }
+        this._useRGBAIfASTCBC7NotAvailableWhenUASTC = value;
+        this._isDirty = true;
+    }
+
+    private _useRGBAIfOnlyBC1BC3AvailableWhenUASTC?: boolean;

```

Comment:
Should this option be true by default? I think it might be better for the default behavior to favor speed instead of memory?